data/json/items/classes/comestible.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "id": "food",
+    "type": "COMESTIBLE",
+    "name": "edible comestible",
+    "comestible_type": "FOOD",
+    "volume": 1,
+    "symbol": "%"
+  },
+  {
+    "id": "fruit",
+    "copy-from": "food",
+    "type": "COMESTIBLE",
+    "name": "base fruit",
+    "material": "fruit",
+    "color": "red",
```

Comment:
That's intentional. The idea being that different groupings of items should have the same colour and symbol?